integration-tests/bats/add-patch-expect/no_then_a.expect

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/expect
+
+set timeout 5
+set env(NO_COLOR) 1
+
+source  "$env(BATS_CWD)/helper/common_expect_functions.tcl"
+
+spawn dolt add -p
+
+# This test will reject the first change for each of the 3 tables, then accept the rest.
+
+expect_with_defaults_2 {| \+ | 0  | Yellow | 255 | 255   | 0    |}    {Stage this row \[y,n,q,a,d,s,\?\]\? } { send "n\r"; }
+
+expect_with_defaults_2 {| - | 2  | Green | 0   | 255   | 0    |}      {Stage this row \[y,n,q,a,d,s,\?\]\? } { send "ar"; }
```

Comment:
Should this one be `send "a\r";`?

Reply:
oh my. good catch. interesting that the test till passes!!!!